data_tools/add.py

```diff
@@ -41,7 +41,31 @@
 if CONTENT_TYPE == "users":
     pass # Not implemented
 elif CONTENT_TYPE == "schemes":
-    pass # Not implemented
+    added = 0
+    skipped_existing = 0
+
+    existing_ids = fcw.get_code_scheme_ids(DATASET_ID)
+    print ("Existing Ids: {}".format(len(existing_ids)))
```

Comment:
Do you think it would read easier if the text is a bit more verbose, like: "{} existing code schemes for this dataset, skipping duplicates"